src/Shell/Output/NullOutputHandler.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Shell\Output;
+
+
+/**
+ * Class NullOutputHandler
+ */
+class NullOutputHandler implements ProcessOutputInterface
+{
+    /**
+     * Handle the command output.
+     *
+     * @param $stdout
+     * @param $stderr
+     * @return void
+     */
+    public function handle($stdout, $stderr)
+    {
+    }
+
+    /**
+     * The stdout read.
+     *
+     * @return string
+     */
+    public function readStdOut()
+    {
+    }
+
+    /**
+     * The stdout read split on newlines.
+     *
+     * @return array
+     */
+    public function readStdOutLines()
+    {
+    }
+
+    /**
+     * The stderr read.
+     *
+     * @return string
+     */
+    public function readStdErr()
+    {
+    }
+
+    /**
+     * The stderr read split on newlines.
+     *
+     * @return array
+     */
+    public function readStdErrLines()
+    {
```

Comment:
Methods declaring they return an array should return an array and not null.

PHP doesn't take kindly to things like `array_map(function ($value) {} ..., null)`. A simple empty array return here will prevent the use of `$handler->readStdErrLines() ?: []` or `$handler->readStdErrLines() ?? []`

Reply:
class was never used thats why there is no implementation => remove